src/Symfony/Component/Translation/Translator.php

```diff
@@ -22,7 +23,7 @@
  *
  * @api
  */
-class Translator implements TranslatorInterface, TranslatorBagInterface
+class Translator implements TranslatorInterface, TranslatorBagInterface, WarmableInterface
```

Comment:
I don't think we should implement WarmableInterface in the component itself,
but just on [Framework Translator](https://github.com/symfony/symfony/blob/2.7/src/Symfony/Bundle/FrameworkBundle/Translation/Translator.php) class